Engine/Strings.resx

```diff
@@ -162,8 +162,17 @@
   <data name="RulesNotFound" xml:space="preserve">
     <value>Cannot find analyzer rules.</value>
   </data>
+  <data name="RuleSuppressionErrorFormat" xml:space="preserve">
+    <value>Suppression Message Attribute error at line {0} in {1} : {2}</value>
+  </data>
+  <data name="RuleSuppressionRuleNameNotFound" xml:space="preserve">
+    <value>Rule {0} cannot be found.</value>
+  </data>
   <data name="StringConstantArgumentsSuppressionAttributeError" xml:space="preserve">
-    <value>All the arguments of the suppression message attribute should be string constants.</value>
+    <value>All the arguments of the should be string constants.</value>
```

Comment:
Oh it's actually gonna be combined with the RuleSuppressionErrorFormat. The logic is in RuleSuppression constructor (last few lines)